pom.xml

```diff
@@ -158,9 +186,53 @@
                 </executions>
             </plugin>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <version>${maven-testing-plugins.version}</version>
+                <groupId>io.confluent</groupId>
+                <artifactId>kafka-connect-maven-plugin</artifactId>
+                <version>${kafka-connect-maven-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>kafka-connect</goal>
+                        </goals>
+                        <configuration>
+                            <title>Datadog Kafka Connect Logs</title>
+                            <name>kafka-connect-logs</name>
+                            <version>${project.version}</version>
+                            <documentationUrl>https://github.com/DataDog/datadog-kafka-connect-logss</documentationUrl>
+                            <description>
+                                A Kafka Connect Connector that sends Kafka Connect records as logs to the Datadog API.
+                            </description>
+                            <logo>logos/datadog.png</logo>
+
+                            <supportProviderName>Datadog</supportProviderName>
+                            <supportSummary>
+                                Support is provided as best effort only. Please register any issues in the github project.
```

Comment:
> Support is provided as best effort only

Do we already have this kind of supportSummary somewhere else?

Reply:
No, but good question, I will ask